### ISSUE TITLE
eagle api

### ISSUE THREAD

**Issue by sneccc** [shingo1228/ComfyUI-send-eagle-slim#2]
Can you give a run down tutorial how eagle api works,
how do i create folders
how do i "upload images"
how do i insert tags, annotations
can u just point me in the code where exactly u do that

**Comment by shingo1228** [comment]
> Can you give a run down tutorial how eagle api works, how do i create folders how do i "upload images" how do i insert tags, annotations can u just point me in the code where exactly u do that

The best way to understand the basic usage of the Eagle API is to read this Eagle API Document. I think it will be more understandable.
[Eagle API Document](https://api.eagle.cool/)

The part of the code for creating folders is here:
https://github.com/shingo1228/ComfyUI-send-eagle-slim/blob/af5f044990c7c9e4916b2303242348878869f540/send_eagle.py#L93-L97

The part of the code for uploading images is here:
https://github.com/shingo1228/ComfyUI-send-eagle-slim/blob/af5f044990c7c9e4916b2303242348878869f540/send_eagle.py#L126

Tags and annotations are in this part of the code. They are added to an item and then sent to Eagle.
https://github.com/shingo1228/ComfyUI-send-eagle-slim/blob/af5f044990c7c9e4916b2303242348878869f540/send_eagle.py#L69-L70

https://github.com/shingo1228/ComfyUI-send-eagle-slim/blob/af5f044990c7c9e4916b2303242348878869f540/prompt_info_extractor.py#L210-L235

**Comment by shingo1228** [comment]
> Also a suggestion , instead of tracking prompt etc , the send to eagle node could have input fields where we just connect the prompts

Thank you. I have previously created a version with input fields, but I decided against adopting it because it was less intuitive to understand how to combine the prompts, which are a hidden attribute, with the text of the input fields, which are a public attribute.

**Comment by shingo1228** [comment]
> Also a suggestion , instead of tracking prompt etc , the send to eagle node could have input fields where we just connect the prompts

Thank you for your suggestion regarding the 'Send to Eagle' node.

After careful consideration, I've decided to revisit the idea of incorporating input fields for connecting the prompts. I'm pleased to announce that I've now developed a new version of the program, named 'Send Eagle with text.' This updated version includes the feature you suggested, allowing users to directly connect prompts via input fields.

I believe this enhancement will make the tool more user-friendly and efficient. I appreciate your feedback and hope this update aligns with your expectations. Feel free to check it out and let me know your thoughts!
